test/whisperkit_test.py

```diff
@@ -0,0 +1,191 @@
+#
+# For licensing see accompanying LICENSE.md file.
+# Copyright (C) 2023 Argmax, Inc. All Rights Reserved.
+#
+
+import os
+import json
+import unittest
+import argparse
+import subprocess
+import concurrent.futures as futures
+
+from pathlib import Path
+from huggingface_hub import hf_hub_download
+from whisper.tokenizer import get_tokenizer
+from android_test_utils import AndroidTestsMixin, TestRunADB
+
+OUTPUT_FOLDER = './json/'
+TEST_DATASET = "librispeech-10mins" 
+TEST_FILES = [
```

Comment:
Can you change this to glob.glob(), sort and select some number of files as appropriate?

Reply:
So the problem is, I'm not trying to load files locally but trying to download a subset of test dataset from here: 
https://huggingface.co/datasets/argmaxinc/whisperkit-test-data/tree/main/librispeech-10mins
From my research, HF API supports either one file download (when I already know the file name) or the entire repo which is way to large. So I used this method inevitably.. 
But I will try to find a more graceful way then this.